packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```
@@ -173,9 +173,659 @@
+          }
+        }
+      },
+      "GetAccountAssetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Asset Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "assetId",
+          "pageSize",
+          "firstTxHash"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "assetId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "integer"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountAssetsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Assets.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "firstAssetId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountDetailRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Detail.",
+        "type": "object",
+        "required": [
+          "pageSize",
+          "paginationKey",
+          "paginationWriter"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "key": {
+            "type": "string"
+          },
+          "writer": {
+            "type": "string"
+          },
+          "pageSize": {
+            "type": "number"
+          },
+          "paginationKey": {
+            "type": "string"
+          },
+          "paginationWriter": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAssetInfoRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Asset Info.",
+        "type": "object",
+        "required": [
+          "assetId"
+        ],
+        "properties": {
+          "assetId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetRolePermissionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Role Permissions.",
+        "type": "object",
+        "required": [
+          "roleId"
+        ],
+        "properties": {
+          "roleId": {
+            "type": "string"
+          }
+        }
+      },
+      "GetEngineReceiptsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Engine Receipts.",
+        "type": "object",
+        "required": [
+          "txHash"
+        ],
+        "properties": {
+          "txHash":{
+            "type": "number"
```

It can contains not only numbers, but also letters, so I suggest:
```suggestion
            "type": "string"
```